# Description: Use the application name for the pulse audio context
# Bug: https://launchpad.net/bugs/1486645
# Author: Olivier Tilloy <olivier.tilloy@canonical.com>

diff --git a/media/audio/pulse/audio_manager_pulse.cc b/media/audio/pulse/audio_manager_pulse.cc
--- a/media/audio/pulse/audio_manager_pulse.cc
+++ b/media/audio/pulse/audio_manager_pulse.cc
@@ -248,7 +248,7 @@
 
   pa_mainloop_api* pa_mainloop_api =
       pa_threaded_mainloop_get_api(input_mainloop_);
-  input_context_ = pa_context_new(pa_mainloop_api, "Chrome input");
+  input_context_ = pa_context_new(pa_mainloop_api, GetGlobalAppName().c_str());
   if (!input_context_)
     return false;
 
